### PR TITLE
updated install script for PipeWire and added more info to Readme

### PR DESCRIPTION
**Pull request by bfenty**
I added a script to install for pipewire and made the changes to the udev rules for pipewire automatic with the script.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ This solves the problem when only **mono** output is recognized for Nari.
 
 ### PipeWire
 
+If you use PipeWire, I've created an install script. Use the install-pipewire.sh script by ``cd`` into the ``razer-nari-pulseaudio-profile`` folder and run ``sudo chmod +x install-pipewire.sh && ./install-pipewire.sh``
+
+Or, you can make the changes yourself manuallY:
```
Comment (imustafin) [inline README.md:15]: Typo here
```suggestion
Or, you can make the changes yourself manually:
```

However, I think this whole line can be removed

File: README.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ This solves the problem when only **mono** output is recognized for Nari.
 
 ### PipeWire
 
+If you use PipeWire, I've created an install script. Use the install-pipewire.sh script by ``cd`` into the ``razer-nari-pulseaudio-profile`` folder and run ``sudo chmod +x install-pipewire.sh && ./install-pipewire.sh``
```
Comment (imustafin) [inline README.md:13]: We can write this a little bit more concisely
```suggestion
If you use PipeWire, use the ``install-pipewire.sh`` script.
```

File: README.md [hunk 3]
```diff
@@ -10,6 +10,10 @@ This solves the problem when only **mono** output is recognized for Nari.
...
+
+Or, you can make the changes yourself manuallY:
+
 If you use PipeWire and you see the default profiles (names like *Multichannel*)
```
Comment (imustafin) [inline README.md:17]: Since you added a paragraph beginning with "If you use PipeWire", does this paragraph become unneeded?

**Review by imustafin — CHANGES_REQUESTED** [review]
Big thanks for the script! I guess it will help a lot.

If possible, can we please make the whole PipeWire section read smoothly as one text?

I added some suggestions, but feel free to rework the whole section.

File: README.md (hunk 1 above)
Comment (bfenty) [reply README.md:15]: Agreed it can be removed. I'm fairly new at this and in fact this is my first pull request lol. Do I make changes or you?

**Comment by bfenty** [comment]
> Big thanks for the script! I guess it will help a lot.
> 
> If possible, can we please make the whole PipeWire section read smoothly as one text?
> 
> I added some suggestions, but feel free to rework the whole section.

Ok, I *think* I added the changes you requested. I think it reads a bit better now.

**Comment by imustafin** [comment]
Looks ok to me now, we can merge this.

 There is no rush, would you like to make it even better? If not, let's merge it as it is. If yes, take you time and improve the text as much as you seem reasonable.

**Pull request merged**